app/src/main/java/co/kr/sopt_seminar_30th/presentation/ui/adapter/ProfileRepositoryAdapter.kt

```diff
@@ -7,25 +7,26 @@ import androidx.recyclerview.widget.DiffUtil
 import androidx.recyclerview.widget.RecyclerView
 import co.kr.sopt_seminar_30th.R
 import co.kr.sopt_seminar_30th.databinding.ItemProfileRepositoryBinding
-import co.kr.sopt_seminar_30th.domain.entity.repository.RepositoryInformation
+import co.kr.sopt_seminar_30th.domain.entity.home.UserRepositoryInformation
+import co.kr.sopt_seminar_30th.domain.entity.tmp.repository.RepositoryInformation
 import co.kr.sopt_seminar_30th.util.MyDiffUtilCallback
 import java.util.*
 
-class ProfileRepositoryAdapter(private val removeData: (RepositoryInformation) -> (Unit)) :
+class ProfileRepositoryAdapter(private val removeData: (UserRepositoryInformation) -> (Unit)) :
     RecyclerView.Adapter<ProfileRepositoryAdapter.HomeRepositoryViewHolder>() {
-    private val itemList = mutableListOf<RepositoryInformation>()
+    private val itemList = mutableListOf<UserRepositoryInformation>()
 
     class HomeRepositoryViewHolder(
         private val binding: ItemProfileRepositoryBinding,
         private val removeItem: (Int) -> (Unit),
-        private val removeData: (RepositoryInformation) -> (Unit)
+        private val removeData: (UserRepositoryInformation) -> (Unit)
```

Comment:
```suggestion
        private val onRemoveData: (UserRepositoryInformation) -> (Unit)
```
콜백 함수는 콜백 함수스러운 이름이 어울립니다

Reply:
함수 이름은 자식 이름처럼